.github/workflows/publish.yml

```diff
@@ -19,15 +19,15 @@ jobs:
         with:
           name: ${{ steps.build-ldp.outputs.artifact-name }}
           path: dist
-      - id: build-lmi
+      - id: build-fhlmi
         uses: hynek/build-and-inspect-python-package@v2
         with:
-          upload-name-suffix: -lmi
+          upload-name-suffix: -fhlmi
       - name: Download built artifact to dist/
         uses: actions/download-artifact@v4
         with:
-          name: ${{ steps.build-lmi.outputs.artifact-name }}
+          name: ${{ steps.build-fhlmi.outputs.artifact-name }}
           path: dist
       - uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          password: ${{ secrets.PYPI_API_TOKEN }}
+          password: ${{ secrets.PYPI_LMI_API_TOKEN }}
```

Comment:
Why do we name this LMI token? Notably this will also publish LDP, so I don't think we should rename this